src/cli/cli.cpp

```diff
@@ -435,6 +445,139 @@ void Interpreter::ProcessHelp(int argc, char *argv[])
     AppendResult(OT_ERROR_NONE);
 }
 
+#if (OPENTHREAD_CONFIG_THREAD_VERSION >= OT_THREAD_VERSION_1_2)
+void Interpreter::ProcessBackboneRouter(int argc, char *argv[])
+{
+    otError                error = OT_ERROR_INVALID_COMMAND;
+    otBackboneRouterConfig config;
+    OT_UNUSED_VARIABLE(argv);
+
+    if (argc == 0)
+    {
+        if (otBackboneRouterGetPrimary(mInstance, &config) == OT_ERROR_NONE)
+        {
+            mServer->OutputFormat("BBR Primary:\r\n");
+            mServer->OutputFormat("server16: 0x%04X\r\n", config.mServer16);
+            mServer->OutputFormat("seqno:    %d\r\n", config.mSequenceNumber);
+            mServer->OutputFormat("delay:    %d secs\r\n", config.mReregistrationDelay);
+            mServer->OutputFormat("timeout:  %d secs\r\n", config.mMlrTimeout);
+        }
+        else
+        {
+            mServer->OutputFormat("BBR Primary: None\r\n");
+        }
+
+        error = OT_ERROR_NONE;
+    }
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+    else
+    {
+        error = ProcessBackboneRouterLocal(argc, argv);
+    }
+#endif
+
+    AppendResult(error);
+}
+
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+otError Interpreter::ProcessBackboneRouterLocal(int argc, char *argv[])
+{
+    otError error = OT_ERROR_NONE;
+    ;
+    otBackboneRouterConfig config;
+    unsigned long          value;
+
+    if (strcmp(argv[0], "disable") == 0)
+    {
+        otBackboneRouterSetEnabled(mInstance, false);
+    }
+    else if (strcmp(argv[0], "enable") == 0)
+    {
+        otBackboneRouterSetEnabled(mInstance, true);
+    }
+    else if (strcmp(argv[0], "jitter") == 0)
+    {
+        if (argc == 1)
+        {
+            mServer->OutputFormat("%d\r\n", otBackboneRouterGetRegistrationJitter(mInstance));
+        }
+        else if (argc == 2)
+        {
+            SuccessOrExit(error = ParseUnsignedLong(argv[1], value));
+            otBackboneRouterSetRegistrationJitter(mInstance, static_cast<uint8_t>(value));
+        }
+    }
+    else if (strcmp(argv[0], "register") == 0)
+    {
+        SuccessOrExit(error = otBackboneRouterRegister(mInstance));
+    }
+    else if (strcmp(argv[0], "state") == 0)
+    {
+        switch (otBackboneRouterGetState(mInstance))
+        {
+        case OT_BACKBONE_ROUTER_STATE_DISABLED:
+            mServer->OutputFormat("Disabled\r\n");
+            break;
+        case OT_BACKBONE_ROUTER_STATE_SECONDARY:
+            mServer->OutputFormat("Secondary\r\n");
+            break;
+        case OT_BACKBONE_ROUTER_STATE_PRIMARY:
+            mServer->OutputFormat("Primary\r\n");
+            break;
+        }
+    }
+    else if (strcmp(argv[0], "config") == 0)
+    {
+        if (argc == 1)
+        {
+            otBackboneRouterGetConfig(mInstance, &config);
```

Comment:
Should also call `otBackboneRouterGetConfig(mInstance, &config);` when `argc > 1`?

Reply:
yes, updated.